src/modules/Builder/features/ActivityItems/ItemConfiguration/InputTypeItems/SliderRows/SliderPanel/SliderPanel.tsx

```diff
@@ -0,0 +1,144 @@
+import { useState, useEffect } from 'react';
+import { useFormContext, FieldValues, Path } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+import { Slider } from '@mui/material';
+
+import { UploaderUiType, Uploader, Table, UiType } from 'shared/components';
+import { InputController } from 'shared/components/FormComponents';
+import { theme, StyledFlexTopCenter } from 'shared/styles';
+
+import { Header } from './Header';
+import { SliderPanelProps } from './SliderPanel.types';
+import {
+  StyledSliderPanelContainer,
+  StyledInputContainer,
+  StyledScoresContainer,
+} from './SliderPanel.styles';
+import { getHeadCells, getTableRows } from './SliderPanel.utils';
+import { SLIDER_VALUE_HEAD_ROWS, SLIDER_LABEL_ROWS } from './SliderPanel.const';
+import {
+  DEFAULT_SLIDER_MIN_NUMBER,
+  SLIDER_LABEL_MAX_LENGTH,
+} from '../../../ItemConfiguration.const';
```

Comment:
pls use abs path

Reply:
do you mean I should change current path to
`from 'modules/Builder/features/ActivityItems/ItemConfiguration.const`
?

Reply:
+